contracts/examples/dao/DAO.sol

```diff
@@ -54,28 +56,20 @@ contract DAO is VentureEth {
         revert("Withdraw is disabled.");
```

Comment:
Do we still need to disable withdraw? If it is `onlyOwner` and the contract is the owner from the first distribution round, it seems to me that no one will be able to execute it.